src/marionette.toolkit.js

```diff
@@ -17,6 +17,16 @@ Toolkit.noConflict = function() {
   return this;
 };
 
+Toolkit.MixinState = function(classDefinition) {
+  let stateMixin = StateMixin;
```

Comment:
so nit picky, but I just had a thought. lowerCase tends to indicate an instance of something.  What do you think about changing this var to `_StateMixin` or something like that?